NAPS2.Escl/Client/EsclClient.cs

```diff
@@ -101,10 +101,15 @@ public async Task<EsclJob> CreateScanJob(EsclScanSettings settings)
         response.EnsureSuccessStatusCode();
         Logger.LogDebug("POST OK");
 
-        return new EsclJob
-        {
-            UriPath = response.Headers.Location!.AbsolutePath
-        };
+        return response.Headers.Location!.IsAbsoluteUri
```

Comment:
Let's do something like `var uri = response.Headers.Location!;` to avoid repetition